cli/crates/server/build.rs

```diff
@@ -39,8 +39,9 @@ Please see the instructions in cli/crates/server/README.md.
 fn decompress_assets() -> io::Result<tempfile::TempDir> {
     use flate2::bufread::GzDecoder;
     let dir = tempfile::tempdir()?;
-    eprintln!("Decompressing the assets at `{ASSETS_GZIP_PATH}` from `server` crate build script.");
-    let file_reader = io::BufReader::new(fs::File::open(ASSETS_GZIP_PATH)?);
+    let assets_gzip_path = env::var("ASSETS_GZIP_PATH").unwrap_or_else(|_| ASSETS_GZIP_PATH.to_owned());
```

Comment:
Might be good to prefix this with grafbase

Reply:
Changed and pushed